docs/introduction-to-product-management.md

```diff
@@ -130,6 +131,40 @@ A common role when the transition to comes from Engineering is the Technical Pro
 
 This is a great opportunity for former engineers that don't have much interest in design or marketing and still want to work very close to technology.
 
+## Skills of a Product Manager
+
+For new entrants in this position, it's hard to devise which skills we need to acquire. In essence, Product Management is a discipline with a broader set of definitions and expectations that makes this career interesting but unclear at the moment of jumping on it.
+ 
+As Cagan said, "It is precisely this combination of deep customer understanding with the ability to apply technology to solve customer problems that enables a strong product person."
```

Comment:
Should we format this as a quote?

Reply:
I'd say yes to be consistent with the rest of the content.